[resources] Add support for the `*_time`; [cryptsetup docs] Add second boot disk to FDE with same passphrase; Minor stuff

ypid opened the pull request.

ypid committed:
[docs] Fix spelling

ypid committed:
[firejail] Ansible 2.9 support

ypid committed:
[docs] cryptsetup: Adding second boot disk to FDE with same passphrase

ypid committed:
[docs] Fix typo

ypid committed:
[resources] Add support for the `access_time` and `modification_time`

drybjed commented on ansible/roles/resources/defaults/main.yml:
```diff
@@ -29,6 +29,15 @@ resources__enabled: True
+# It is set here to a human readable profile of `ISO 8601`_ because the Ansible
+# default format is hardly human readable
+# (without separators like ``-`` and ``:``).
+resources__time_format: '%Y-%m-%dT%H:%M:%S'
```
I'm not sure what the time format is used for. Does that indicate how Ansible should interpret the string provided by the user? In that case, perhaps you should add an example or two of the usage with a date specified in the desired format, in the variable documentation.

ypid replied:
> Does that indicate how Ansible should interpret the string provided by the user?

That is correct. And the Ansible default format is totally unreasonable and not very human readable.

I checked the variable docs but you decided to refer to the Ansible module docs which I find very reasonable. Therefore I documented it in the `defaults/main.yml`. I now added a example to it as well.

drybjed replied:
No, I meant a proper example in the variable documentation in `docs/ansible/roles/resources/defaults-detailed.rst`, sorry for the misunderstanding. The small example near the variable definition can stay.

ypid replied:
Done that. Should be ready for merging.

ypid committed:
[resources] No special ISO 8601 profile is being used

The idea was to maybe use %Y-%m-%dt%H:%M:%S (lower case `t`) instead for
"enhanced" human readability but I decided against it to be very ISO
8601 complaint.

ypid committed:
[resources] Add timestamp example for the Ansible file module

ypid committed:
[cryptsetup] Improve second boot disk to FDE with same passphrase docs

ypid committed:
[cryptsetup] Update docs, mention LUKS2

ypid committed:
[tinyproxy] Coding style: Fix indention of default variable

ypid committed:
[etesync] Mention that pip is used to install etesync for now

ypid committed:
[cryptsetup] Improve spelling in docs

ypid committed:
[resources] Detailed docs for `access_time` and `modification_time`

drybjed commented:
@ypid: Almost there. :-) Got the error from the docs on Travis-CI:
```
/home/travis/build/debops/debops/docs/ansible/roles/cryptsetup/defaults-detailed.rst:599:Duplicate explicit target name: "cryptsetup__ref_devices_chaining_multiple_ciphers".
```

ypid committed:
[docs] Fix copy paste mistake

ypid commented:
Fixed :)

drybjed reviewed (approved):
Looks good, thanks. I'll backport selected commits to older DebOps releases.
